src/zitadel.rs

```diff
@@ -47,16 +47,46 @@ impl Zitadel {
 			zitadel_client,
 		})
 	}
+
 	/// Import a list of new users into Zitadel
 	pub(crate) async fn import_new_users(&self, users: Vec<User>) -> Result<()> {
 		for user in users {
 			let zitadel_user =
 				user.to_zitadel_user(&self.feature_flags, &self.zitadel_config.idp_id);
-			let sync_status = self.import_user(&zitadel_user).await;
+			let status = self.import_user(&zitadel_user).await;
 
-			if let Err(error) = sync_status {
-				tracing::error!("Failed to sync user `{}`: {:?}", zitadel_user.log_name(), error);
-			};
+			if let Err(error) = status {
+				tracing::error!(
+					"Failed to sync-import user `{}`: {:?}",
+					zitadel_user.log_name(),
+					error
+				);
+
+				if Self::is_invalid_phone_error(error) {
+					let zitadel_user = ZidatelUser {
```

Comment:
Looks like we missed this in the previous review, this struct is named `ZidatelUser` :D We should fix that, but I don't think it needs to be this PR as I'd say it's ready to merge.

Reply:
I don't quite follow, the name seems appropriate?

Reply:
It's "Zidatel" instead of "Zitadel"

Reply:
Good catch! I couldn't see it either 😂